tests/test_vfpproperties.cpp

```diff
@@ -46,9 +46,9 @@
 #include <opm/parser/eclipse/EclipseState/Tables/VFPProdTable.hpp>
 #include <opm/parser/eclipse/Units/UnitSystem.hpp>
 
-#include <opm/autodiff/VFPProperties.hpp>
```

Comment:
Would it not be better to have the tests deal with the non-ADB version of the VFP code, as that is more likely to live long and prosper?

Reply:
sure. that would be a quite bit more work, though?

Reply:
You are right. We should test the non-ADB code. I will make an update.

Reply:
Fixed in new PR